config/core/test-configs.yaml

```diff
@@ -1640,6 +1640,11 @@ device_types:
             - 'tinyconfig'
             - 'kvm_guest.config'
 
+  xilinx-zcu102:
```

Comment:
This device type doesn't appear to be in any lab.  However it is defined in LAVA upstream.

There is this instead in lab-cip:
https://lava.ciplatform.org/scheduler/device_type/zynqmp-zcu102

Is this the same device type but with a different name?

Reply:
yes is this one.

Reply:
fixed the name